index.js

```diff
@@ -27,6 +28,12 @@ function init(editor, onSave) {
 	}
 
 	editor.setCursorBufferPosition(cursorPosition);
+
+	if (editor.getScreenLineCount() < line){
```

Comment:
The intent would probably also be clearer if this case where inverted and line 36 put into this block.